extrabees/src/main/java/binnie/extrabees/utils/config/ConfigurationMain.java

```diff
@@ -6,7 +6,7 @@
 
 	private static final String WORLDGEN = "World-Gen";
 	public static boolean canQuarryMineHives = true;
-	public static int waterHiveRate = 1;
+	public static int waterHiveRate = 2;
```

Comment:
The default value as indicated in line 19
`waterHiveRate = config.getInt("waterHiveRate", WORLDGEN, 2, …`
is also 2 so any other value would be irritating here.